### PR TITLE
Merge decorators and unified class features proposals

### PR DESCRIPTION
**Pull request by littledan**
These were always just one proposal; "unified class features" was intended as a proposed update to the decorators proposal. I've confirmed this with champions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,14 +27,13 @@ This list contains only stage 1 proposals and higher that have not yet been with
 |   | [Optional catch binding][optional-catch]                         | Michael Ficarra                                 | 3 |
 |   | [`function.sent` metaproperty][function-sent]                    | Allen Wirfs-Brock                               | 2 |
 |   | [`String.prototype.{trimStart,trimEnd}`][trims]                  | Sebastian Markbage                              | 2 |
-|   | [Class and Property Decorators][decorators]                      | Yehuda Katz and Brian Terlson                   | 2 |
+|   | [Class and Property Decorators][decorators] ([update](unified-class)) | Yehuda Katz and Brian Terlson                   | 2 |
```
Comment (alangpierce) [inline README.md]: This is a broken link. I think it should be `([update][unified-class])`.

Comment (littledan) [reply README.md]: Thanks, fixed

**Comment by littledan** [comment]
Sorry for my lack of clarity in the presentation.

**Comment by ljharb** [comment]
For me it was mainly that it's a separate repo; I'd have expected an update to decorators to be on the decorators repo.

**Pull request merged**